ts/store/actions/instabug.ts

```diff
@@ -0,0 +1,12 @@
+/**
+ * Action types and action creator related to InstabugInfo.
+ */
+import { ActionType, createStandardAction } from "typesafe-actions";
+
+export const instabugUnreadMessagesLoaded = createStandardAction(
+  "RESPONSE_INSTABUG_INFO_LOADED"
```

Comment:
I guess this name is too generic and it could be confusing
https://github.com/teamdigitale/io-app/pull/1369/files#r336111616